tests/test_ui.py

```
@@ -613,48 +513,72 @@ def test_render_table_view_nan_std():
         steps._render_table_view(["S1"])
 
 
-def test_render_footer_reset_hit_proper_mock():
-    """Properly mock columns and button to hit reset logic."""
+def test_render_footer_reset_hit_proper_mock(mock_streamlit_columns):
```

Ruff (0.15.12)

[warning] Unused function argument: `mock_streamlit_columns`

(ARG001)